src/main/scala/is/hail/expr/Relational.scala

```diff
@@ -3,9 +3,9 @@ package is.hail.expr
 import java.io.FileNotFoundException
```

Comment:
Is it possible for these changes to be a separate PR depending on the changes that add `OrderedRDD2`?